includes/class-wc-cart.php

```diff
@@ -51,6 +51,13 @@ class WC_Cart extends WC_Legacy_Cart {
 	public $applied_coupons = array();
 
 	/**
+	 * Contains an array of discount objects applied to the cart.
+	 *
+	 * @var array
+	 */
+	protected $manual_discounts = array();
```

Comment:
Should we just call these 'discounts' vs 'manual discounts'? Coupons have unique naming.